clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ConsumerSubscriptionCleaner.java

```diff
@@ -156,10 +161,13 @@ Set<PubSubTopicPartition> getTopicPartitionsToUnsubscribe(
 
     recordNumberOfTopicsToUnsub.accept(topicsToUnsubscribe.size());
 
+    Set<PubSubTopicPartition> topicPartitionsForNonAliveDataReceiver =
```

Comment:
Actually we don't need to do this here, we can just to data receiver liveness check in for loop (L#168+). This way we can avoid double iterations

Reply:
It also looks like this might unintentionally remove some subscriptions from the consumer task.

For example, suppose:
	•	test_store_rt-0 is ingesting data for SIT_v1
	•	test_store_rt-1 is ingesting data for SIT_v2

If SIT_v1 is no longer alive, we might mistakenly remove the subscription for test_store_rt-1 as well due to the check in the later part of the code.

Reply:
After discussion, we decided to revert the change for data receiver temporarily, as we plan to refactor the the clearner in future PR.